can/bus.py

```diff
@@ -141,4 +143,13 @@ def shutdown(self):
         """
         self.flush_tx_buffer()
 
-    __metaclass__ = abc.ABCMeta
+    @property
+    @abc.abstractmethod
+    def state(self):
+        return 'Should never read this'
+
+    @state.setter
+    def state(self, new_state):
+        pass
+
+    __metaclass__ = abc.ABCMeta
```

Comment:
nitpick: missing newline

Reply:
We do try to! But over many years with many developers things are hard to keep "normal".